app/services/process_parser.rb

```diff
@@ -24,8 +24,10 @@ class ProcessParser
 
   ##
   # @param [Nokogiri::XML::Element] process_element
-  def initialize(process_element)
+  # @param [Boolean] provide default as lane_id if no lane_id
+  def initialize(process_element, use_default_lane_id = true)
```

Comment:
Would you mind making this a kwarg?  I find this makes the code much easier to read.

Reply:
I do not mind. Fixed.